Mathlib/Data/List/Range.lean

```diff
@@ -42,33 +43,43 @@ theorem chain_range_succ (r : ℕ → ℕ → Prop) (n a : ℕ) :
   rw [range_succ_eq_map, chain_cons, and_congr_right_iff, ← chain'_range_succ, range_succ_eq_map]
   exact fun _ => Iff.rfl
 
+-- Verify that `finRange` is semireducible.
+example : finRange 3 = [0, 1, 2] := rfl
```

Comment:
Can this go directly after `def finRange`, or are the numerals not available there?

Reply:
Done.